mxcube3/__init__.py

```diff
@@ -82,7 +82,7 @@ def complete_initialization(app):
         app.session = app.beamline.getObjectByRole("session")
         app.collect = app.beamline.getObjectByRole("collect")
         app.diffractometer = app.beamline.getObjectByRole("diffractometer")
-        if not hasattr(app.diffractometer, 'centring_motors_list'):
+        if getattr(app.diffractometer, 'centring_motors_list') is None:
```

Comment:
The mockup is wrong. There is a `__getattr__` that always returns `None` if attribute is
missing, see those lines in `DiffractometerMockup.py`:

```
    # to make it compatibile
    def __getattr__(self, attr):
        if attr.startswith("__"):
            raise AttributeError(attr)
        else:
            if attr == "currentCentringProcedure":
                return self.current_centring_procedure
            if attr == "centringStatus" :
                return self.centring_status
```

It would be great to add `raise AttributeError(attr)` at the end.

Meanwhile, the solution could be to replace `hasattr` test by `try...except AttributeError`
like this:

```
    try:
        app.diffractometer.centring_motors_list
    except AttributeError:
        # my code in the 'if'
        ...
```

What do you think ?